### ISSUE TITLE
Create AWS Lambda that sends a message to another user

### ISSUE THREAD

**Issue by heiheihang** [LalaCuteGroup/lala-referral#7]
We need to create an AWS lambda that facilitates the communication between applicants and referrers. 

The AWS Lambda should take the `applicationId` of the *referral*, and it should automatically infer that *User A* wants to send a message to *User B*

**Comment by jonowo** [comment]
How to distinguish between the cases where applicant sends message to referrer and referrer sends message to applicant?

**Comment by jonowo** [comment]
I forgot that the Lambda will also take the message from the user as input. ඞඞඞ

Ideally there should be an additional way to send a message which does not involve using commands. It can be achieved like this:
1. A tells the bot to send a message to B
2. The bot sends a message M to B containing the content sent by A
3. B is able to send a response to A by replying to M directly without using a command
4. Repeat with A and B swapped

This also handles the situation of talking to multiple applicants/referrers, eliminating the users' need to keep track of application IDs.

However the bot will need to know which user to relay the message to when the applicant ID is not specified.
- One solution is to create an additional table to store the application ID, ID of receiving user and message ID (the last 2 uniquely identify a message in any chat) associated with each message relayed by the bot.
- A lazier solution would be to include the application ID in the content of any relayed message sent (which we should be doing anyway), and then extract it to be used afterward when the relayed message is being replied to.

ඞඞඞ

**Comment by heiheihang** [comment]
I agree with your suggestion. Suggestion 2 makes sense as we are likely to display the application ID to the applicant anyway.
